CI/JenkinsfileTestCleanInstance

```diff
@@ -58,331 +57,348 @@ pipeline {
-      image_name = "polycube"
-      registryCred = credentials('polycube-repo')
-      notification_url = credentials('slack-notifications')
-      github_token = credentials('github_status_update')
+        image_tag = "${env.GIT_COMMIT}"
+        image_name = "polycube"
+        registryCred = credentials('polycube-repo')
+        notification_url = credentials('slack-notifications')
+        github_token = credentials('github_status_update')
     }
-
     stages {
-
-    stage ('Build Base Image') {
-    agent {
-             label "docker"
-    }
-     steps {
-        script {
-                 commit_id = sh returnStdout: true, script: "git rev-parse HEAD"
-                 setGithubStatus("${commit_id}", "pending", github_token, "polycube-network/polycube", "Same Instance Tests", "Same Instance Tests")
-                 setGithubStatus("${commit_id}", "pending", github_token, "polycube-network/polycube", "Iptables Tests", "Iptables Tests")
-                 setGithubStatus("${commit_id}", "pending", github_token, "polycube-network/polycube", "Clean Instance Tests", "Clean Instance Tests")
-                 setGithubStatus("${commit_id}", "pending", github_token, "polycube-network/polycube", "Docker Image Build", "Docker Image Build")
-                 slackNotification("STARTED",notification_url,"Polycube")
-                 docker.withRegistry("", 'polycube-repo') {
-                 sh """export DOCKER_BUILDKIT=1
-                     docker system prune --all --force
-                     docker build . -t "polycubebot/base_image:latest" -f Dockerfile_base_image
-                     docker push polycubebot/base_image:latest
-            """
+        stage ('Init Notification Handlers') {
+            agent {
+              label "docker"
+            }
+            steps {
+                script {
+                    commit_id = sh returnStdout: true, script: "git rev-parse HEAD"
+                    setGithubStatus("${commit_id}", "pending", github_token, "polycube-network/polycube", "Same Instance Tests", "Same Instance Tests")
+                    setGithubStatus("${commit_id}", "pending", github_token, "polycube-network/polycube", "Iptables Tests", "Iptables Tests")
+                    setGithubStatus("${commit_id}", "pending", github_token, "polycube-network/polycube", "Clean Instance Tests", "Clean Instance Tests")
+                    setGithubStatus("${commit_id}", "pending", github_token, "polycube-network/polycube", "Docker Image Build", "Docker Image Build")
+                    slackNotification("STARTED",notification_url,"Polycube")
+                }
             }
         }
-      }
-     post {
-        failure {
-          script {
-            setGithubStatus("${commit_id}", "pending", github_token, "polycube-network/polycube", "Docker Image Build", "Docker Image Build")
-          }
+        stage('Build Base Image') {
+            when {
+                triggeredBy 'TimerTrigger'

```

Comment:
the branch should also be master otherwise it will built at 2am for every branch